cmd/create/network/templates/rosa-quickstart-default-vpc/cloudformation.yaml

```diff
@@ -23,9 +28,13 @@ Parameters:
     Default: '10.0.0.0/16'
 
 Conditions:
-  HasAZ1: !Equals [!Ref AvailabilityZoneCount, 1]
-  HasAZ2: !Equals [!Ref AvailabilityZoneCount, 2]
-  HasAZ3: !Equals [!Ref AvailabilityZoneCount, 3] 
+  AZ3Explicit: !Equals [Fn::Length: !Ref AvailabilityZones, 3]
```

Comment:
Availability Zone could be 4 ex; us-west-2 , can we consider 4 AZs as well ?

Reply:
Yes, this is possible. But in a separate PR.